en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+# Gimbal Protocol (v2)
+
+> **Note** This version is marked as work-in-progress.
+  This means that it is still subject to change.
+
+<span></span>
+> **Note** This version supersedes [Gimbal Protocol v1](../services/gimbal.md)
+
+## Introduction
+
+Gimbal protocol v2 was developed due to [various issues](../services/gimbal.md#known_issues) with the version 1.
```

Comment:
End users don't care at this point why we developed this version - or at least it is a minor interest. 

What we need here is "why we need a gimbal protocol", "what use cases does it cover". At the end you can have a note with a link to the design doc saying something like "this service was developed as a result of known issues in the previous version. For information about the design see [link to design doc].

The sorts of things we would say are somewhat covered - we need to manage multiple sources of inputs (e.g. from missions, nudge from controller, etc) and we have gimbals with vastly different capabilities. The goal of the gimbal protocol is to provide a consistent interface for services that use a gimbal and a clear interface layer that can target gimbals with different hardware and software capabilities. (or similar :-)